website/docs/reference/operators/array-operators.md

```diff
@@ -65,11 +65,15 @@ db.team.insertMany([
 
 ## $all
 
-*Syntax*: `{ <field>: { $all: [ <value1>, <value2>, ... ] } }`
+*Syntax*: `{ <field>: { $all: [ <element1>, <element2>, ... <elementN> ] } }`
 
-The `$all` operator matches documents where the field value is an array that contains all the specified elements.
+The `$all` operator matches documents where a particular field is an array containing all the elements given in the query.
```

Comment:
`$all` operator selects documents with an array field that contains all elements from the given query.

Some suggestion, not exact wording.